contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.1.0--4.2.0.sql

```diff
@@ -31,6 +31,26 @@ end
 $$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
```

Comment:
this also should not be required. only 4.0--4.1 is needed to handle upgrades

Reply:
Actually, if a user is on extension version 4.3/4.4 (and so on) and trying to go to 5.x then they may face the same error.